tests/test_math.py

```diff
@@ -752,6 +752,30 @@ def test_matrix_44():
         assert "Matrix44 should only be a 2D array of shape (4,4)." in str(val.value)
 
 
+def test_create_translation_matrix():
+    vector = Vector3D([1, 2, 3])
+    expected_matrix = Matrix44([[1, 0, 0, 1], [0, 1, 0, 2], [0, 0, 1, 3], [0, 0, 0, 1]])
+    translation_matrix = Matrix44.create_translation(vector)
+    assert np.array_equal(expected_matrix, translation_matrix)
+    assert translation_matrix.is_translation()
+
+
+def test_create_rotation_matrix():
+    direction_x = Vector3D([1, 0, 0])
+    direction_y = Vector3D([0, 1, 0])
+    expected_matrix = Matrix44([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]])
+    rotation_matrix = Matrix44.create_rotation(direction_x, direction_y)
+    assert np.array_equal(expected_matrix, rotation_matrix)
+
+    direction_x = Vector3D([1, 0, 0])
+    direction_y = Vector3D([0, 1, 0])
+    direction_z = Vector3D([0, 0, 1])
+
+    expected_matrix = Matrix44([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]])
+    rotation_matrix = Matrix44.create_rotation(direction_x, direction_y, direction_z)
+    assert np.array_equal(expected_matrix, rotation_matrix)
+
```

Comment:
Can we do a more "complex" test? The current rotation is basically non-existent because your X, Y and Z vectors are the same as the global coordinate system.